### PR TITLE
Sebastian/Money update

### PR DESCRIPTION
**Pull request by fjavier-moreno**
Add, among others, getConvertedAmount method.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
         "doctrine/doctrine-cache-bundle": "~1.0",
         "mmoreram/simple-doctrine-mapping": "~0.1",
 
-        "sebastian/money": "~1.3",
+        "sebastian/money": "1.5.*",
```
Comment (mmoreram) [inline composer.json]: Does Sebastian's Money implementation promises BC Break?

Comment (fjavier-moreno) [inline composer.json]: Sebastian does not comment any incompatibility with older versions.

Comment (mmoreram) [inline composer.json]: I am talking about BC Break Promise. That means that you can use `~1.5` with any fear of BC Break.

Comment (fjavier-moreno) [inline composer.json]: I'm talking about this update only add new methods. Not change any previous methods used by Elcodi and add some news like getConvertedAmount, that is very useful when you work with API's and Twig extension is not valid to format php output.

Comment (mmoreram) [inline composer.json]: I know. Just saying that if you put `1.5.*` then you cannot use `1.6.*`, and this is not the behavior we want.

If you want this change to be merged, please use the `~` symbol -> `~1.5`

**Comment by alch** [comment]
@fjavier-moreno @mmoreram I don't think there will be a breaking change for a given minor version `1.5.*`
Anyway why not stick with the `~1.3` version and then accept the PR?

**Comment by xphere** [comment]
If we rely on `getConvertedAmount` for anything, we need to upgrade dependency to `~1.5`.
If this is project-specific, then there's no need for that in elcodi nor bamboo.

**Comment by mmoreram** [comment]
@fjavier-moreno why did you close this PR?

**Comment by jromero86** [comment]
You how to calculate discounts if given a number with more than two decimal then?

**Pull request closed without merging**